src/main/java/kafdrop/service/KafkaMonitorImpl.java

```diff
@@ -116,27 +116,31 @@ public ClusterSummaryVO getClusterSummary(Collection<TopicVO> topics) {
   }
 
   private Map<String, TopicVO> getTopicMetadata(String... topics) {
-    final var topicInfos = highLevelConsumer.getTopicInfos(topics);
-    final var retrievedTopicNames = topicInfos.keySet();
-    final var topicConfigs = highLevelAdminClient.describeTopicConfigs(retrievedTopicNames);
-
-    for (var topicVo : topicInfos.values()) {
-      final var config = topicConfigs.get(topicVo.getName());
-      if (config != null) {
-        final var configMap = new TreeMap<String, String>();
-        for (var configEntry : config.entries()) {
-          if (configEntry.source() != ConfigSource.DEFAULT_CONFIG &&
-              configEntry.source() != ConfigSource.STATIC_BROKER_CONFIG) {
-            configMap.put(configEntry.name(), configEntry.value());
-          }
-        }
-        topicVo.setConfig(configMap);
-      } else {
-        LOG.warn("Missing config for topic {}", topicVo.getName());
-      }
-    }
-    return topicInfos;
-  }
+		final var topicInfos = highLevelConsumer.getTopicInfos(topics);
```

Comment:
I think there are still some issues with your formatter setup. Before raising the PR have a look at the 'Files changed' tab to see how your code looks once meged.

Reply:
I am formatting using eclipse default formatter. Is any thing specific I should consider. Let me c check at Files changed tab